SETI/rms-opus
# 1153 update import to handle new pdsfile changes, make sure NHxxMV has thumbnail images

- Fixes #1153, need to work with https://github.com/SETI/pds-webtools/pull/37
- Were any Django, import pipeline, table_schema, or dictionary files modified? Y
  - If YES:
    - Database used: re-import after updating viewables rules in NHxxxx_xxxx in pds-webtools
    - All Django tests pass: Y
    - FLAKE8 run on modified code: Y
- Were any JavaScript or CSS files modified? N
  - If YES:
    - JSHINT run on all affected files: NA
    - Tested on Chrome, Firefox, Safari, Edge, and iOS: NA
      (Remember to test all browser sizes)
    - Tested for race conditions (with delayed API calls if applicable): NA

Description of changes:
- Add api/images.json tests .
- Make sure test script can create api/images.json automatically.
Known problems:
